src/site/content/en/how-to-optimize-inp/diagnose-slow-interactions-in-the-lab/index.md

```diff
@@ -117,18 +117,18 @@ To get more information on why an interaction is slow, you'll need to take thing
 When the profile populates, the first place to look should be the activity summary at the top of the profiler. The activity summary will show red bars at the top where long tasks occurred in the recording. This allows you to quickly zoom in on problem areas.
 
 <figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/BuE3dMMHux2xLl1jm40k.png", alt="A screenshot of the activity summary in the performance panel of Chrome DevTools. The activity displayed is mostly from JavaScript that causes a long task, which is highlighted in red above the flame chart.", width="573", height="87" %}
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/UcSCVSfw65rg6u1YkqmY.png", alt="A screenshot of the activity summary in the performance panel of Chrome DevTools. The activity displayed is mostly from JavaScript that causes a long task, which is highlighted in red above the flame chart.", width="674", height="126" %}
   <figcaption>
     The activity summary at the top of Chrome's performance profiler. Long tasks are highlighted in red above the activity flame chart. In this case, significant scripting work was responsible for most of the work in the long task.
   </figcaption>
 </figure>
 
-You can quickly focus on the problem area by dragging and selecting a region in the activity summary. Once you've focused to where the interaction occurred, the interactions track will help you line up the interaction and the activity that occurred in the main thread track below it:
+You can quickly focus on the problem area by dragging and selecting a region in the activity summary. Once you've focused to where the interaction occurred, the **Interactions** track will help you line up the interaction and the activity that occurred in the main thread track below it:
 
 <figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/taWMcW9SqVCufYcfpOQc.png", alt="A screenshot of an interaction as visualized in the performance panel of Chrome DevTools. An interactions track above the main thread track shows the duration of an interaction, which can be lined up with main thread activity.", width="800", height="424" %}
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/YKYLTuog3110ynQyXDdM.png", alt="A screenshot of an interaction as visualized in the performance panel of Chrome DevTools. An interactions track above the main thread track shows the duration of an interaction, which can be lined up with main thread activity.", width="800", height="325" %}
```

Comment:
The [old diagram](https://web.dev/diagnose-slow-interactions-in-the-lab/#:~:text=You%20can%20quickly%20focus%20on%20the%20problem%20area%20by%20dragging%20and%20selecting%20a%20region) was a bit clearer in differentiating pointer down and pointer up, and that they were part of the same interaction id (1753)

Reply:
Didn't agree with this one? Or just not possible to do this on new panel?